src/constants/pools.ts

```diff
@@ -484,6 +484,26 @@ const POOLS_MAP = {
   [Network.POLYGON]: POOLS_POLYGON,
   [Network.ARBITRUM]: POOLS_ARBITRUM,
 };
+
+export const POOLS_NAMES_MAP: Record<string, string> = {
+  '0x7b50775383d3d6f0215a8f290f2c9e2eebbeceb20000000000000000000000fe':
+    'Balancer Boosted Aave USD',
+  '0x8fd162f338b770f7e879030830cde9173367f3010000000000000000000004d8':
+    'Balancer Boosted Aave USD',
+  '0x06df3b2bbb68adc8b0e302443692037ed9f91b42000000000000000000000063':
+    'Balancer Stable USD',
+  '0xd387dfd3a786e7caa06e6cf0c675352c7ffff30400000000000000000000063e':
+    'Balancer Stable USD',
+  '0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014':
+    'Balancer Protocol Liquidity',
+  '0xdc2ecfdf2688f92c85064be0b929693acc6dbca6000200000000000000000701':
+    'Balancer Protocol Liquidity',
+  '0xf8a0623ab66f985effc1c69d05f1af4badb01b00000200000000000000000060':
+    'Balancer Protocol Liquidity',
+  '0x3dd0843a028c86e0b760b1a76929d1c5ef93a2dd000200000000000000000249':
+    'AuraBAL Stable Pool',
+};
+
```

Comment:
Please make this network specific. i.e. update the `Pools` type to have a `Names` attribute. Then in each of the networks add the Names attribute along with any relevant mappings.

Reply:
Updated